### ISSUE TITLE
Update master to use _updated

### ISSUE THREAD

**Issue by dskoda1** [mjhea0/flask-boilerplate#18]
I see that you have a newer version of the app layout in `_updated`, which is more modularized and lends itself to not having super long files. Is there a reason why master hasn't been migrated to that version? 

You could create a release with the current code base and then enforce the `_updated` folder upon master. I could open a PR with these changes if this is something that would be accepted.

**Comment by mjhea0** [comment]
did you see this branch?

https://github.com/mjhea0/flask-boilerplate/tree/updated

thoughts?

**Comment by dskoda1** [comment]
Yep, that is exactly what I am talking about. I did not see it before raising this issue as you can tell however 😄  I still would propose what I said before about saving the current state as a release and making master reflect the updated changes. Theres only a few other boilerplates out there and while I wasn't thrilled with the master branch the `updated` looks great! I think people could really get some use with it.
